### PR TITLE
chmod: update Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution.

@kant I have applied the changes to match the page with English page, since I am not a native speaker I have left the descriptions as it is instead of using machine translation tools. Can you add the translations for the 2 descriptions?

**Review by kant — COMMENTED** [review]
LGTM

**Comment by kant** [comment]
> @kant are you still working on this PR?

yes, Sir

File: pages.es/common/chmod.md [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
 
-`chmod g-x {{archivo}}`
+`chmod g-x {{ruta/al/archivo}}`
 
 - Otorga a todos los usuarios (a) derechos para leer y ejecutar:
 
-`chmod a+rx {{archivo}}`
+`chmod a+rx {{ruta/al/archivo}}`
 
 - Otorga a [o]tros (que no están en el grupo del propietario) los mismos derechos que los del grupo:
```
[the remark this answers is not in the document]
Comment (kant) [reply pages.es/common/chmod.md:22]: LGTM

**Review by kbdharun — APPROVED** [review]

**Pull request merged**